pandas/tests/extension/test_decimal.py

```diff
@@ -0,0 +1,143 @@
+import decimal
+import numbers
+import random
+import sys
+
+import numpy as np
+import pandas as pd
+import pandas.util.testing as tm
+import pytest
```

Comment:
see my comments above, you need another level here, otherwise this is really really messy.